### ISSUE TITLE
The pretrained STME module

### ISSUE THREAD

**Issue by imyzx2017** [qiaott/MirrorGAN#7]
Did you re-trained the STME module based on 'attnGAN' ? If so, could you provided it? thanks!

After adding my own pretrained caption model, replacing the DAMSM loss to caption_loss doesn't improve the IS, can you give some advices?

**Comment by mkumar10** [comment]
I think they add it with the DAMSM loss not replace it. Then you should be able to reproduce.

**Comment by saramahersalem** [comment]
@imyzx2017  Hello @imyzx2017 , could you help me to train model to my own caption dataset?

**Comment by rafafelixphd** [comment]
@qiaott in the paper you have experimented on CUB-Birds. However, in this repo you have added STREAM model only for COCO. Can you possibility release the weights for CUB-BIRDS data set as well?

Best wishes, Rafa.
